backend/apis/clients/follow-ups/create-follow-up.api.js

```diff
@@ -32,7 +32,9 @@ app.post("/api/clients/:clientId/follow-ups", (req, res) => {
     ...checkValid(req.body, nullableValidDateTime("appointmenetDate")),
   ];
 
-  const { clientId } = req.params;
+  let { clientId } = req.params;
+
+  clientId = Number(clientId);
```

Comment:
URL params always come into express as strings, even though the clientId should be a number. Using it as a string wasn't causing issues, but I fixed this when I noticed it when logging sql queries.